llm/ggml.go

```diff
@@ -171,7 +279,9 @@ func DecodeGGML(r io.ReadSeeker) (*GGML, error) {
 	}
 
 	model, err := c.Decode(&ro)
-	if err != nil {
+	if errors.Is(err, io.EOF) {
+		// noop
```

Comment:
What is the case that causes this error, and why is it ok?

Reply:
Unlike GGUF which knows how much it needs to read, GGLA reads until EOF at which point it returns